test/test_helper.rb

```diff
@@ -16,9 +16,9 @@
   # Use knapsack to split up tests on CI nodes
   # To rerack the test divider, run:
   # KNAPSACK_GENERATE_REPORT=true bundle exec rake test test:system
-  require 'knapsack'
-  knapsack_adapter = Knapsack::Adapters::MinitestAdapter.bind
-  knapsack_adapter.set_test_helper_path(__FILE__)
+  # require 'knapsack'
```

Comment:
CircleCI seems to have built a lot more around parallelization since we last looked - see https://circleci.com/docs/2.0/parallelism-faster-jobs/ . Knapsack may not be necessary anymore.